Documentation/teaching/index.rst

```diff
@@ -70,6 +70,11 @@ tools/labs/docs/Dockerfile for dependencies):
    labs/memory_mapping.rst
    labs/device_model.rst
 
+.. toctree::
+   :caption: Assignments
+
+   so2/assign0-kernel-api.rst
+
```

Comment:
The assignment needs to be added only to the SO2 section. Remove this.

Reply:
Fixed.